lib/fizzy/utf8.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "types.hpp"
```

Comment:
I don't think this is needed here.

Reply:
It was, for `bytes_view`.

Reply:
About that, it would be nice to move `bytes` / `bytes_view` out of `types.hpp`. Using `types.hpp` suggests this file depends on wasm types.

Reply:
> About that, it would be nice to move bytes / bytes_view out of types.hpp. Using types.hpp suggests this file depends on wasm types.

Yep, I was about to do that like 5 times already :)